### DISCUSSION TITLE
Bug Report: Source code version crashes after choosing any week

### DISCUSSION THREAD

**Discussion by kriggirsedicc** [KadeArchive/Kade-Engine#396] [Q&A]
### If you are playing a downloaded version of the game, what operating system are you using?
Windows (`x86`), Windows (`x86_64`), Linux, or macOS? Specify below.

## What version of the game are you using? Look in the bottom left corner of the main menu.

EDIT: I meant Kade Engine 1.4.2, FNF version was 2.7.1

## Have you identified any steps to reproduce the bug? If so, please describe them below. Use images if possible.

Build game with Lime in PowerShell and then select any week.

## Please describe your issue. Provide extensive detail and images if possible.

So I decided to get into modding and I've done a few basic stuff like adding my own character, but whenever I build the game, it launches perfectly and all, but whenever I select a week it completely closes and shows this in PowerShell: 

source/ModchartState.hx:76: haxe value not supported - (x: 0 | y: 0) which is a type of TClass(flixel.math.FlxPoint)

I am extremely new to this, so cut me some slack.

**Comment by puyoxyz** [comment]
Converting to discussion since this happens after you modded so it's not an issue with KE itself
